### PR TITLE
docs(repo): update node version in contributing guidelines

### LINKED ISSUE #2227: Possibly outdated node version in contributing guideline
Current [contributing guidelines](https://github.com/stoplightio/elements/blob/main/CONTRIBUTING.md) suggest that node v12 should be used, however project contains dependencies that require newer version, which can be misleading. Some of the packages are:

- `@stoplight/elements-core@7.6.5` is set to `"node": ">=14.13"`
- `@stoplight/json-schema-viewer@4.7.1` is set to `"node": ">=16"`

## Context

Better contributing guidelines

## Current Behavior

Using node version suggested by guidelines doesn't work

## Expected Behavior

Contributing guidelines suggest correct node version

### PR DESCRIPTION
**Pull request by heitortsergent**
Closes #2227.

I also ran CONTRIBUTING through Grammarly and fixed some grammar issues.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,43 +62,43 @@ For your convenience all the packages are linked. So, for example, if you run `e
...
 - The goal for your test suite is **to cover** as much of the **business requirements** (e.g. in the issue description) as practical.
 - An ideal test suite only requires a change **if business requirements change**.
 - While not generally expected, **it is OK to add additional tests** for helper functions, hooks and subcomponents where it would not be practical to test them alongside their host components.
 In this case, still try to follow the principles above as much as practical.
-- Use **Jest/JSDOM testing wherever suitable**. Cypress-based tests should be reserved for high level integration tests, such as:
+- Use **Jest/JSDOM testing wherever suitable**. Cypress-based tests should be reserved for high-level integration tests, such as:
     - Basic sanity checks to make sure that *Elements* builds and loads correctly in a certain environment.
-    - Making sure that the routers/navigation of the external framework play nicely with our internal router.
+    - Making sure that the routers/navigation of the external framework plays nicely with our internal router.
```
Comment (dotslashderek) [inline CONTRIBUTING.md:86]: I would just make this "make sure the routers of the external framework play nicely with our internal router."

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -216,48 +217,49 @@ Elements is used in the Stoplight Platform, as well as in open source projects.
...
-6. Now make a push, create a PR and ask someone for a review. The purpose of the review is to make sure the release obeys our versioning guidelines.
+5. After the script gets finalized, a commit will be created for you.
+6. *IMPORTANT!!!* If `elements-core` version was updated in `elements` and/or `elements-dev-portal`, the script has changed `~` sign to `^` in front of `elements-core` version. Change it back to `~` in both places manually and make a new commit. The commit message doesn't really matter here, something like `fix: ^ to ~ for elements-core` is more than enough.
+7. Now make a push, create a PR and ask someone for a review. The purpose of the review is to make sure the release obeys our versioning guidelines.
 
 ## Versioning Guidelines
 
-If you did changes only in `elements` package, it's okay to release only `elements` package.
+If you did changes only in the `elements` package, it's okay to release only the `elements` package.
```
Comment (dotslashderek) [inline CONTRIBUTING.md:232]: Instead of "did changes" I would use "made changes" (here and in the next couple of lines)

**Review by dotslashderek — APPROVED** [review]
Looks good a couple suggestions but reads nice as is - thanks!

File: CONTRIBUTING.md (hunk 2 above)
Comment (heitortsergent) [reply CONTRIBUTING.md:232]: Oh great catch! I completely missed that 🤦 .

**Pull request merged**